### PR TITLE
Add docs for Deno workspaces

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: runtime/manual/basics/workspaces/index.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+| version            | ❌        | ✅      |                                                                                                                                                                                                                                                                                        |
+| exports            | ❌        | ✅      |                                                                                                                                                                                                                                                                                        |
+| workspace          | ✅        | ❌      | Nested workspaces are not supported.                                                                                                                                                                                                                                                   |
+
+### Specyfing workspace configuration
```
Comment (lambtron) [inline runtime/manual/basics/workspaces/index.md:267]:
```suggestion
### Specifying workspace configuration
```

**Review by lambtron — APPROVED** [review]
LGTM (so far)

**Comment by bartlomieju** [comment]
Don't land. `.docusaurus` needs to be removed first.

**Review by dsherret — APPROVED** [review]
LGTM

**Pull request merged**